content/join.md

```diff
@@ -12,45 +12,90 @@ even host in-person breakout rooms.  Read more about [how to attend a
+**We recommend that you form your own team and do exercises together
+in your own breakout room (in-person or online)**.  We will support
+this as much as we can!  You *will* be able to ask us questions if you
+register.
 
-- Register as type **Livestream only**
+**Please register anyway so that you can get emails and we can better
+report our impact** (but you don't have to).
 
-The promise of the Internet is that we can reach everyone, so why
-don't we?  So, we have a live stream
-([twitch.tv/coderefinery](https://twitch.tv/coderefinery)) option so
-that anyone may attend
-the workshop, even if we don't have resources to provide interactive
-breakout rooms.  The live stream option is also good if you want to
-passively see what we are talking about, but don't want to take an
-interactive Zoom seat.  **Please register anyway so that you can get
-emails and we can report our impact**, but you do not have to.
 
-**We recommend that you form your own team and do
-exercises together in your own breakout room (in-person or online)**.
-We will support this as much as we can!  You *will* be
-able to ask us questions if you register.
+
+### Attend via livestream + Zoom
+
+To provide interactive support, we also host parallel a Zoom session.
+Learners watch the livestream during teaching, and work in Zoom during
+exercises.
+
+In Zoom, we group attendees into teams with an exercise leaders and
+help from our staff.  This makes a very interactive workshop, but
+unfortunately we have limited resources to accept these
+registrations.
+
+- Register as type ``Zoom Learner`` or ``Zoom Exercise leader``
+- You may register as a **solo learner**, we'll try to take as many as
+  we can (up to the number of exercise leaders who volunteer)
+- If a **team** registers with their own exercise leader, you get
+  priority and we will try to accept the whole team.  Everyone on the
+  team should register separately and set the same `Team name` in the
+  registration.
+
+Attending via Zoom will give you everything the livestream does, with
+the possibly of centrally organized teams and a bit more help.
+
+
+
+### FAQ: how should I register?
+
+- **I am alone and want to work with others, how should I register?**
+  - Register for Zoom (if we are full you get the livestream anyway)
+- **Why teams?**
+  - People learn best in small groups, even if the primary teaching
+    sessions can reach many people and be effective.  Our strategy
+    gives us the best of both worlds.
+- **I am making my own team, how should I register?**
+  - **Want more personal support:** `Zoom` you will have more
+    support from our expert helpers, which may be good for less
+    confident teams.
+  - **Want to manage everything yourself** (for example make your own
+    Zoom or in-person room): `Livestream only`.  The coordinator can
+    still join our CodeRefinery chat and interact with us.
+  - The coordinator could register on behalf of your team and forward
```

Comment:
isn't this contradictory with the point below, which encourages everyone to register so that CR can better measure the environment?

Reply:
Is it better now?  I tried to do some sort of progressive disclosure, make it simple then make it more complex.  But yeah, it's not good if it's inconsistent.  Is this good?

Reply:
yes I think now it's good